### PR TITLE
Test i18n translation package

### PR DESCRIPTION
**Pull request by alexchoiweb**
Test for Issue #116 

-add message translate capability in `<BeforeYouBegin>`
-add language toggler nav to `<App>`
-add translated message files in public/locales (message was first manually translated by using Google Translate)

https://user-images.githubusercontent.com/35116846/113346286-7d77c000-92e8-11eb-9039-2bf5a561b951.mov

### CODE REVIEW

File: react/client/public/locales/en/translation.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "disclaimerText": {
+    "0": "Please make sure you are eligible for getting your record cleared. If you need help with this please email placeholder@website.com.",
+    "1": "Be aware your statement is stored locally on your device, and never transmitted to our servers or any third-party",
+    "2": "This software is developed by a non-governmental third party, Code for America.",
+    "3": "Please also review our Privacy Policy and Terms of Use before beginning.",
+    "4": "Please allocate at least 30 minutes of time to complete this.",
+    "5": "If at anytime you are confused please click the question mark button for guidance."
+  },
```
Comment (aahvocado) [inline react/client/public/locales/en/translation.json:9]: i have a suspicion that we can still do it on one line and use a newline character `\n` and have it work (might need to style it with `white-space: pre-line;`). The only awkward thing is translators would need to recognize the newline but we won't concern ourselves too much about it right now.

File: react/client/public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "disclaimerHeader": {
+    "0": "Before you begin"
+  }
```
Comment (aahvocado) [inline react/client/public/locales/en/translation.json:12]: I get why you had to use numbers for access keys on the above one but ideally we can be descriptive with our keys - maybe this one could be `disclaimer.headerText`

File: react/client/src/App.tsx [hunk 3]
```diff
@@ -42,12 +44,31 @@ const theme = createMuiTheme({
...
+                padding: '2rem 0',
+                backgroundColor: 'gray',
+                display: 'flex',
+                justifyContent: 'center',
+              }}
+            >
+              <button onClick={() => handleClick('en')}>ENG</button>
+              <button onClick={() => handleClick('esp')}>ESP</button>
+              <button onClick={() => handleClick('ko')}>KO</button>
+            </nav>
```
Comment (aahvocado) [inline react/client/src/App.tsx:70]: A little language dropdown, neat!

File: react/client/public/locales/en/translation.json (hunk 2 above)
Comment (alexchoiweb) [reply react/client/public/locales/en/translation.json:12]: Great idea, done

File: react/client/src/App.tsx (hunk 3 above)
Comment (alexchoiweb) [reply react/client/src/App.tsx:70]: Yes, i'll comment it out for now until design team figures out how they want to implement it, if they want to implement it

File: react/client/public/locales/en/translation.json (hunk 1 above)
Comment (alexchoiweb) [reply react/client/public/locales/en/translation.json:9]: Beautiful, it works, thanks, that's a new one for me

**Review by aahvocado — APPROVED** [review]

**Pull request merged**